examples/pytorch/eeg-gcnn/README.md

```diff
@@ -0,0 +1,55 @@
+# DGL Implementation of EEG-GCNN Paper
+This example is a simplified version that presents how to utilize the original EEG-GCNN model proposed in the paper [EEG-GCNN](http://proceedings.mlr.press/v136/wagh20a.html), implemented with DGL library. The original code is [here](https://github.com/neerajwagh/eeg-gcnn).
+
+## All References
+- [ML4H Poster](https://drive.google.com/file/d/14nuAQKiIud3p6-c8r9WLV2tAvCyRwRev/view?usp=sharing) can be helpful for understanding data preprocessing, model, and performance of the project. 
+- The recording of presentation by the author Neeraj Wagh can be found on [slideslive](https://slideslive.com/38941020/eeggcnn-augmenting-electroencephalogrambased-neurological-disease-diagnosis-using-a-domainguided-graph-convolutional-neural-network?ref=account-folder-62123-folders).
+- The slides used during the presentation can be found [here](https://drive.google.com/file/d/1dXT4QAUXKauf7CAkhrVyhR2PFUsNh4b8/view?usp=sharing).
+- Raw Data can be found in these two links: [MPI LEMON](http://fcon_1000.projects.nitrc.org/indi/retro/MPI_LEMON.html) (no registration needed), [TUH EEG Abnormal Corpus](https://www.isip.piconepress.com/projects/tuh_eeg/downloads/tuh_eeg_abnormal/) ([needs registration](https://www.isip.piconepress.com/projects/tuh_eeg/html/request_access.php))
+
+## Dependencies
+
+- Python 3.8.1
+- PyTorch 1.7.0
+- DGL 0.6.1
+- numpy 1.20.2
+- Sklearn 0.24.2
+
+## Dataset
+- Final Models, Pre-computed Features, Training Metadata can be downloaded through [FigShare](https://figshare.com/articles/software/EEG-GCNN_Supporting_Resources_for_Reproducibility/13251452).
+- In EEGGraphDataset.py, we specify the channels and electrodes and use precomputed spectral coherence values to compute the edge weights. To use this example in your own advantage, please specify your channels, electrodes and generate your own spectral coherence values. All example datasets can be downloaded from FigShare.
+## How to Run
+First, download psd_features_data_X, labels_y, spec_coh_values.npy, and standard_1010.tsv.txt. Put them in the repo. <br>
```

Comment:
Use `figshare_upload/psd_features_data_X` and `figshare_upload/labels_y` instead.

Reply:
I saw `spec_coh_values.npy` and `standard_1010.tsv.txt` in both `psd_shallow_eeg-gcnn` and `psd_deep_eeg-gcnn`. Are they the same?

Reply:
Yes, they are the same.